client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/hooks/use-pattern-pages.ts

```diff
@@ -99,7 +99,7 @@ const usePatternPages = (
 		.filter( Boolean ) as Pattern[];
 
 	pagesToShow = pageCategoriesInOrder
-		.map( ( category: Category ) => {
+		?.map( ( category: Category ) => {
```

Comment:
I cannot reproduce this. This is impossible because `ReferenceError` is only thrown when the variable does not exist, whereas this `pageCategoriesInOrder` is clearly defined in the scope.

Maybe there was a hiccup when Calypso was hot-reloading?

Could you reproduce this consistently?

Reply:
I cannot reproduce it anymore. I think it was a hot-reloading issue.